queue_model_objects_v1.py

```diff
@@ -241,11 +241,9 @@ def get_display_name(self):
             return ''
 
     def init_from_sc_sample(self, sc_sample):
-        #self.loc_str = ":".join(map(str,sc_sample[-1]))
-        #self.location = sc_sample[-1]
-        #self.set_name(self.loc_str)
-        self.loc_str = str(sc_sample[1]) + ':' + str(sc_sample[2])
-        self.location = (sc_sample[1], sc_sample[2])
+        self.loc_str = ":".join(map(str,sc_sample[-1]))
```

Comment:
Clear, I will test it and look how it works. I guess MAXIV people could look at this before the PR is accepted. Second problem is that at EMBL we use master branch and other use 2.1, 2.2. So we should find time and possibility to merge all branches.